scripts/deploy.js

```diff
@@ -0,0 +1,29 @@
+const envsub = require('envsub')
```

Comment:
Of course JS has a dependency for this already 😂

Reply:
I would use envsubst but I didn't want to do the iso date in bash, and json validation and minification is easier in js too, so there we are 😅.